### ISSUE TITLE
Dreamcatcher locket quest

### ISSUE THREAD

**Issue by rockfireredmoon** [rockfireredmoon/iceee#396]
Reported By: Misha Moon
Category: QUEST

The quest in Deadwood, taken from Morrigan Shadowstar to find a chest with a locket inside has a marker that leads to Bleeding Sap, but the chest is actually in the chapel in the first quest area of Grunes Tal

**Comment by LiskaQuicksilver** [comment]
To clarify this bug, the chest that is giving the reward should not even be related to that quest at all as it is intended to be a NPC reward chest for the Iron Maw dungeon quest. The real chest containing that Locket is supposed to be in the Kazechek camps across from Ironbirch Camp on the ground in those ruins and supposed to spawn randomly between them. The chest should be in the same locations that one also finds the mob Shromdar. My guess is this is yet another bit of data corruption from the VS rollout problems. Should be an easy fix.

**Comment by rockfireredmoon** [comment]
The player reported problem could not be reproduced. The chest appears in the correct place and is collectable by the player. The map marker is a little off, but it is a "search" quest so this is expected and right. Please either retest to confirm. I am going to guess t his was originally caused by the quest file corruption.

**Comment by LiskaQuicksilver** [comment]
As it turns out, the marker was not the only issue with this quest. Testing it out, the marker is in the general area where it supposed to be and the quest can be completed as normal. That said, I decided to investigate the chest in Grunes Tal that the player claimed they could collect the quest item from.

Sure enough, the Iron Chest sitting in Redeemer's Hall in Grunes Tal is in fact the same one used for The Lost Locket quest. Moreover, because of this, if the quest is active, the player can actually complete the quest objective in this way out of the way location.

My guess is, when we were still building Grunes Tal, I borrowed that chest as a placeholder with the intent to replace it with the proper item later, then we all forgot what it went to and left it as is. Best solution I can come up with is to create a Chest for Shar Zaris to offer rewards from and leave it empty until we get to finishing up The Iron Maw dungeon as his chest was meant to hold rewards for the player upon completion of that dungeon. At least, this was the original concept I had in mind way back when. If you would prefer to simply disable the item for now, that is fine as well.
